manual.md

```diff
@@ -52,25 +51,25 @@ The Radworks governance process is defined by three seperate phases: Discussion,
 
 Each phase has specific timelines and requirements:
 
-| Discussion | Formal Review | Submission |
+| Request for Comments (RFC) | Formal Review | Submission |
 | --- | --- | --- |
 | Forum post | Snapshot poll | On-chain vote |
 | ≥ 7 days | 7 day | 3 days vote, 2 day timelock  |
 
-RAD Governance Proposals (RGPs) must be formally labeled with the phase of the proposal, #, and title (e.g. [Discussion/Formal Review/Submission][RGP - #] - [PROPOSAL TITLE])
+RAD Governance Proposals (RGPs) must be formally labeled with the phase of the proposal, #, and title (e.g. [RFC/Formal Review/Submission][RGP - #] - [PROPOSAL TITLE])
 
-## **Discussion**
+## **Request for Comments (RFC)**
 
-The _Discussion_ phase is used to review the first draft of new proposals. This is the primary period for the community to review and provide feedback on proposals drafts. The feedback provided during this phase allows for more detailed and refined proposals to vote on in later stages.
+The _Request for Comments_ phase is used to actively review formal proposal drafts that are being submitted to governance within a given cylce. While drafts of proposals can be posted on the forum _at any time_, RFC is the official period for the community to review and provide feedback on proposals that will be voted on in a given cylce. The feedback provided during this phase allows for more detailed and refined proposals to vote on in later stages.
 
-In order for a draft proposal to be formally considered in a [proposal cycle](#proposal-cycles), it must be posted as a _Discussion_ using one of the templates linked below. 
+In order for a draft proposal to be formally included in a [proposal cycle](#proposal-cycles), it must be labeled with [RFC] and formatted using the templates linked below. 
 
-When preparing the proposal drafts for the _Discussion_ phase, proposal authors should:
-- Use [these templates](https://github.com/radicle-foundation/radworks-governance/tree/main/templates) to create their proposals. Please see the 'Proposal Types' section below to determine which template to use. 
-- Try to include as much detail as possible in this first draft of their proposals
-- Post their proposal under [Proposal Drafts](https://community.radworks.org/c/governance/discussions/19) on the forum by the _second Monday_ of each cycle (see 'Proposal Cycles' for details).  
+> 💡 For more complex proposals, it is encouraged to post early drafts in advance of the intended proposal cycle to allow for ample discussion and debate. Authors are encouraged to specify the expected or tentative proposal cycle for their submission.
```

Comment:
Should we add a note about what category these should be posted to on Discourse?